src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -154,17 +154,29 @@ subctl join --kubeconfig cluster-a/auth/kubeconfig broker-info.subm --clusterid
 ```
 
 ####  Verify Deployment
-To verify the deployment follow the steps below:
+To manually verify the deployment follow the steps below.
 
 ```bash
-kubectl --kubeconfig cluster-b/auth/kubeconfig create deployment nginx --image=nginx
-kubectl --kubeconfig cluster-b/auth/kubeconfig expose deployment nginx --port=80
-kubectl --kubeconfig cluster-a/auth/kubeconfig run --generator=run-pod/v1 tmp-shell --rm -i --tty --image nicolaka/netshoot -- /bin/bash
-curl nginx
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 create deployment nginx --image=nginx
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment nginx --port=80
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 -n default apply -f - <<EOF
+apiVersion: lighthouse.submariner.io/v2alpha1
+kind: ServiceExport
+metadata:
+  name: nginx
+EOF
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash curl nginx.default.svc.supercluster.local:8080
```

Comment:
```suggestion
kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash curl -4 nginx.default.svc.supercluster.local:8080
```

Let's avoid the silly curl bug (just in case)